javascript-carousel/main.js

```diff
@@ -0,0 +1,55 @@
+var $image = document.querySelector('.image');
+var $dots = document.querySelectorAll('.fa-circle');
+var $leftArrow = document.querySelector('.fa-chevron-left');
+var $rightArrow = document.querySelector('.right');
+
+var currentImageArrayIndex = 0;
+
+var imageArray = [
+  './images/001.png', './images/004.png', './images/007.png', './images/025.png', './images/039.png'
+];
+
+function changeImageHandler(event) {
+  var whichArrow = event.target.getAttribute('data-view') || 'right';
+  if (whichArrow === 'left' && currentImageArrayIndex === 0) {
```

Comment:
I think you can vastly simplify this by breaking up the functionality for the arrows into two functions, one to move left, and one to move right.